packages/lodestar-cli/src/commands/dev/command.ts

```diff
@@ -146,7 +147,8 @@ export class DevCommand implements ICliCommand {
       logger: new WinstonLogger({module: "API"}),
       chain: node.chain,
       network: node.network,
-      db: node.db
+      db: node.db,
+      reps: new ReputationStore()
```

Comment:
I'm pretty sure this is useless as you need reputation store from beacon node itself